### DISCUSSION TITLE
Error notice in hplm()

### DISCUSSION THREAD

**Discussion by MoHerzog** [jazznbass/scan#74] [Q&A] [answered]
Hello, I have a specific question on an error notice.

When running a hplm (N=10) with one additonal variable ("HYP") including interactions with level and random slope effects:

"hplm(mod1,update.fixed=.~ mt+phaseB+interB+HYP*phaseB,data.l2 = L1,trend=TRUE, slope=TRUE, level=TRUE, random.slopes = TRUE)"

follwing error notice is returned:

"Fehler in lme.formula(fixed = fixed, random = random, data = dat, na.action = na.omit,  : 
  optim problem, convergence error code = 1
  message =" 

How do I interpret the error notice - does the hplm simply not converge? And how can I address this problem?

Thanks in advance!

**Comment by jazznbass** [comment, marked as answer]
Hello Moritz,
you have too few data points (cases) for a stable estimate of the variance of the random slope. 
You can try one or more or all of the following:
- omit the random slope parameter,
- drop the trend effect
- drop the slope Effect
- increase the iterations of the convergence function (add to hplm: `control = nlme::lmeControl(maxIter = 1e8, msMaxIter = 1e8, opt = "optim")`)

**Reply by MoHerzog** [reply]
Hello Jürgen,

thank you very much! Increasing the number of iterations worked perfect 👍 

Best wishes,
Moritz
